packages/app/src/app/pages/Dashboard/Content/SandboxCard/index.tsx

```diff
@@ -217,6 +218,13 @@ class SandboxItem extends React.PureComponent<Props, State> {
             },
             color: theme.red.darken(0.2)(),
           },
+          {
```

Comment:
Could you put this one above the  `Move to Trash` with a separator? So it  would be:

```
Privacy controls
---
Export
---
Move to trash
```